web-ng/src/app/services/data-store/data-store.service.ts

```diff
@@ -54,6 +54,23 @@ export class DataStoreService {
       );
   }
 
+    /**
+   * Returns an Observable that loads and emits the projects with the specified
+   * uuid.
+   *
+   */
+  loadAllProject$() {
+      return this.db
+        .collection('projects')
+        .snapshotChanges().pipe(
+          map(actions => actions.map(a => {
+            const data = a.payload.doc.data() as Project;
+            const id = a.payload.doc.id;
```

Comment:
is a null check required here ? I don't understand this codebase enough to know.

Reply:
TypeScript has strict nullness checking, so if the compiler isn't complaining it's _probably_ ok.